### PR TITLE
Add user story

### PR DESCRIPTION
**Pull request by bigludo7**
#### What type of PR is this?

Add one of the following kinds:

* documentation

#### What this PR does / why we need it:

Add user story

#### Which issue(s) this PR fixes:

Fixes #

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

File: documentation/API_documentation/OTPValidationAPI_User_Story.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+| ---- | ------- |
+| ***Summary*** | As an enterprise application developer, I want to verify the phone number associated with the line from which the call was made, so I can get a proof of possession of the phone number.  |
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User, Customer:BusinessManager, Customer:Administrator<br> **Actors:** Application service providers, hyperscalers, application developers, end users. <br> **Scope:**  <br> - Customer:user asks CSP to send a message featuring a 'code' variable via SMS. The message is specified by Customer:user. In the API response a request identifier is provided to link with validation step. |
+| ***Pre-conditions*** |The preconditions are listed below:<br><ol><li>The Customer:BusinessManager and Customer:Administrator have been onboarded to the CSP's API platform.</li><li>The Customer:BusinessManager has successfully subscribed to the OTP Validation API product from the product catalog.</li><li>The Customer:Administrator has onboarded the Customer:User to the platform.</li><li>The Customer:user performs an authorization request to CSP</li><li>The Customer:user has the access token alllowing a secure access of the API.|
```
Comment (jgarciahospital) [inline documentation/API_documentation/OTPValidationAPI_User_Story.md:11]: alllowing --> allowing

File: documentation/API_documentation/OTPValidationAPI_User_Story.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User, Customer:BusinessManager, Customer:Administrator<br> **Actors:** Application service providers, hyperscalers, application developers, end users. <br> **Scope:**  <br> - Following a send code request, customer:user asks end user to keyed the code received by SMS. Then customer:user send this code to the CSP to validate that the code matches the one send by the CSP in the SMS.  |
+| ***Pre-conditions*** |The preconditions are listed below:<br><ol><li>The Customer:BusinessManager and Customer:Administrator have been onboarded to the CSP's API platform.</li><li>The Customer:BusinessManager has successfully subscribed to the OTP Validation API product from the product catalog.</li><li>The Customer:Administrator has onboarded the Customer:User to the platform.</li><li>The Customer:user performs an authorization request to CSP</li><li>The Customer:user has the access token alllowing a secure access of the API.</li><li> The Customer:user has performed a valid send code previously and have the request identifier (authenticationId). |
```
Comment (jgarciahospital) [inline documentation/API_documentation/OTPValidationAPI_User_Story.md:23]: Question: Include action end-user receives SMS code and provides to Customer:user application

**Review by jgarciahospital — CHANGES_REQUESTED** [review]

**Comment by bigludo7** [comment]
Thanks @jgarciahospital 
All your comments considered in new proposal.

File: documentation/API_documentation/OTPValidationAPI_User_Story.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# OTPVAlidationAPI API User Story
+
+OTP Validation user story is splitted in 2 steps. Both steps are mandatory to perform  this user story.
```
Comment (fernandopradocabrillo) [inline documentation/API_documentation/OTPValidationAPI_User_Story.md:3]:
```suggestion
OTP Validation user story is splitted in 2 steps. Both steps are mandatory to perform this user story.

```

**Review by jgarciahospital — APPROVED** [review]

**Review by fernandopradocabrillo — APPROVED** [review]
LGTM

**Pull request merged**